src/rgw/rgw_auth_s3.cc

```diff
@@ -571,7 +571,7 @@ get_v4_canonical_headers(const req_info& info,
     }
     const char* const t = info.env->get(token_env.c_str());
     if (!t) {
-      dout(10) << "warning env var not available" << dendl;
+      dout(10) << "warning env var not available " << token_env.c_str() << dendl;
```

Comment:
what does token_env contain here?  will this make sense when printed?

Reply:
It contains the header name that has been in the signedheaders field but the RGW didn't get it